{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/globals.py

```diff
@@ -0,0 +1,144 @@
+####################
+# Project metadata #
+####################
+
+PROJECT_NAME = '{{cookiecutter.package_name}}'
+CLUSTER_PROJECT = 'proj_cost_effect'
+
+CLUSTER_QUEUE = 'all.q'
+MAKE_ARTIFACT_MEM = '3G'
+MAKE_ARTIFACT_CPU = '1'
+MAKE_ARTIFACT_RUNTIME = '3:00:00'
+
+LOCATIONS = [
+    # TODO - project locations here
+]
+
+
+#############
+# Data Keys #
+#############
+
+METADATA_LOCATIONS = 'metadata.locations'
+
+POPULATION_STRUCTURE = 'population.structure'
+POPULATION_AGE_BINS = 'population.age_bins'
+POPULATION_DEMOGRAPHY = 'population.demographic_dimensions'
+
+ALL_CAUSE_CSMR = 'cause.all_causes.cause_specific_mortality_rate'
+
+# TODO - sample disease keys
+DIARRHEA_CAUSE_SPECIFIC_MORTALITY_RATE = 'cause.diarrheal_diseases.cause_specific_mortality_rate'
```

Comment:
I think instead of examples, we want a description of how to write the keys here and their relationship to the EntityKey class for the artifact, for now.  We'll talk about setting up autogeneration of these keys in the next sprint or the one following.

Reply:
The description should include a couple of examples, but not all from diseases.

Reply:
We might also link to the vivarium inputs data pulling tutorial which explains what measures are available for entity types.

Reply:
Do this in this PR